### DISCUSSION TITLE
Bat does not rewrap long lines after resizing terminal

### DISCUSSION THREAD

**Discussion by jdiamond** [sharkdp/bat#2450] [Q&A] [answered]
Hi, if I use less or bat (which uses less from what I read in the README) to view a file that has long lines, they wrap at the edge of the terminal as they should. When I maximize my terminal, less seems to automatically rewrap everything so the long lines end up getting redrawn to use the new wider terminal. With bat, this does not happen. After resizing, the newly available space on the right is just empty. I am using bat 0.19.0 on Ubuntu 22.04 inside WSL2. My terminal is the new one that comes with Windows (it's just called "Terminal"?). I don't know much about configuring terminals so not sure if this is related to bat but since it works with plain less and not with bat, I thought I would look for help here first. Thanks.

**Comment by keith-hall** [comment, marked as answer]
I don't think there are any solutions for this except to configure `bat` to disable `bat`'s line wrapping (and grid): https://github.com/sharkdp/bat/issues/322#issuecomment-625976828
